src/ui/server/controllers/insurance/policy/single-contract-policy/index.test.ts

```diff
@@ -103,9 +103,9 @@ describe('controllers/insurance/policy/single-contract-policy', () => {
             ID: CONTRACT_COMPLETION_DATE,
             ...FIELDS.CONTRACT_POLICY.SINGLE[CONTRACT_COMPLETION_DATE],
           },
-          POLICY_CURRENCY_CODE: {
-            ID: POLICY_CURRENCY_CODE,
-            ...FIELDS.CONTRACT_POLICY[POLICY_CURRENCY_CODE],
+          CURRENCY_CODE: {
```

Comment:
we are using this and then mapping to `POLICY_CURRENCY_CODE`.  Is it not worth using `POLICY_CURRENCY_CODE `here instead?

Reply:
But then it would be unaligned with all currency form fields. I think all currency form fields should use the same ID, `CURRENCY_CODE`?